S/solar/bundled/patches/windows.patch

```diff
@@ -0,0 +1,11 @@
+--- a/makefile
++++ b/makefile
+@@ -1,7 +1,7 @@
+ EXE                = ../bin/solar
+ COMPILATOR         = g++
+ COMPILATOR_OPTIONS = -O3 -Wall -Wextra -pedantic -Wcast-align
+-LIBS               = -lc -lm
```

Comment:
Without patching you can also call `make LIBS="-lm"`

Reply:
I don't know what `-lc` do '^^

Reply:
Links to the c standard library, which is kinda weird and unnecessary for C++ code.

Reply:
If it's not needed, we can remove thé patch and call `make LIBS="-lm"`

Reply:
Yes, that's what I meant above :slightly_smiling_face: